src/toil/lib/aws/utils.py

```diff
@@ -40,7 +41,7 @@
                             retry)
 
 if sys.version_info >= (3, 8):
-    from typing import Literal
+    from typing import Literal, MutableMapping
 else:
     from typing_extensions import Literal
```

Comment:
```suggestion
    from typing import Literal
else:
    from typing_extensions import Literal
from typing import MutableMapping
```

[The `typing` module has MutableMapping all the way back to 3.7](https://docs.python.org/3.7/library/typing.html#typing.MutableMapping), so we can always get it from there, not just when we're on Python 3.8 or newer. And we can't just not import it when we're on 3.7, since we actually need it.